x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/plan/logical/command/sys/SysColumns.java

```diff
@@ -185,6 +185,10 @@ private static Object odbcCompatible(Integer value, boolean isOdbcClient) {
         }
         return value;
     }
+    
+    private static boolean isOdbcClient(Mode mode) {
```

Comment:
No need to add a method for a simple check that is only used once.

Reply:
This method is used twice actually. [Here](https://github.com/elastic/elasticsearch/pull/39518/files#diff-8dd83d03e8d8b05d260314f0b99c40d0R100) as well.

Reply:
My point still stands - it's a basic check.